libs/cardano-ledger-pretty/src/Cardano/Ledger/Pretty/Conway.hs

```diff
@@ -203,12 +176,53 @@ instance PrettyA GovernanceActionIx where
 instance PrettyA VoterRole where
   prettyA = ppString . show
 
+instance PrettyA (GovernanceMetadata era) where
+  prettyA = viaShow
+
 instance PrettyA (PParamsUpdate era) => PrettyA (GovernanceActionState era) where
-  prettyA GovernanceActionState {..} =
+  prettyA (GovernanceActionState vs d ra act en) =
```

Comment:
It is good that you want to protect against forgetting a field when one is added, but you don't need to loose readability of `RecordWildCards`. In fact it is better to retain RWC, because renaming fields should also be reflected in pretty and compiler will notify when that happens, so you can do this instead:
```suggestion
  prettyA gas@(GovernanceActionState {..}) =
    let GovernanceActionState _ _ _ _ _ = gas -- ensure we do not forget to add/remove fields
    in
```